src/components/backend-ai-session-launcher.ts

```diff
@@ -2001,14 +2017,67 @@ export default class BackendAiSessionLauncher extends BackendAIPage {
     render(
       html`
         <vaadin-text-field id="vfolder-alias-${rowData.item.name}" clear-button-visible prevent-invalid-input
-                           pattern="^[a-zA-Z0-9\._-]*$" ?disabled="${!rowData.selected}"
-                           theme="small" placeholder="${rowData.item.name}"
+                           pattern="^[a-zA-Z0-9\./_-]*$" ?disabled="${!rowData.selected}"
+                           theme="small" placeholder="/home/work/${rowData.item.name}"
                            @change="${(e) => this._updateFolderMap(rowData.item.name, e.target.value)}"></vaadin-text-field>
         </template>
       `,
       root
     );
   }
+
+  infoHeaderRenderer(root, column?) {
+    render(
+      html`
+      <div class="horizontal layout center">
+        <span id="vfolder-header-title">${_t('session.launcher.FolderAlias')}</span>
+        <mwc-icon-button icon="info" class="fg green info" @click="${(e) => this._showPathDescription(e)}"></mwc-icon-button>
+      </div>
+      `,
+      root
+    );
+  }
+
+  _showPathDescription(e?) {
+    if (e != undefined) {
+      e.stopPropagation();
+    }
+    this._helpDescriptionTitle = _text('session.launcher.FolderAlias');
+    this._helpDescription = _text('session.launcher.DescFolderAlias');
+    this._helpDescriptionIcon = '';
+    const pathDialog = this.shadowRoot.querySelector('#help-description');
+    setTimeout(() => this.setPathContent(pathDialog));
+    pathDialog.show();
+  }
+
+  setPathContent(pathDialog) {
+    const pathLastChild = pathDialog.children[pathDialog.children.length - 1];
+    const pathContentLastChild = pathLastChild.children[pathLastChild.children.length - 1];
+    if (pathContentLastChild.children.length < 6) {
```

Comment:
Let's not use the hard-coded numbers only if it doesn't need any explanation. How about assigning value to a new variable so that others could understand your code in future.

Reply:
Edit done!